Switch admin UI to Vuetify 💄 🙃 🐐 👔

underscope opened the pull request.

underscope committed:
Vuetify ContentModal :bowtie:

underscope committed:
Vuetify imported repo list

underscope committed:
Vuetify EnrollmentModal :wrench:

underscope committed:
Vuetify Enrollment list

underscope committed:
Vuetify Cohort root :wrench:

underscope committed:
Removed cohort list page :wrench:

underscope committed:
Vuetify CohortModal :wrench:

underscope committed:
Vuetify Navbar :wrench:

underscope committed:
Vuetify Sidebar :wrench:

underscope committed:
Vuetify UserModal :wrench:

underscope committed:
Vuetify user list :wrench:

underscope committed:
Vuetify root container

underscope committed:
Imported Vuetify

underscope committed:
Cleanup :lipstick:

vladimyr commented:
>![image](https://user-images.githubusercontent.com/1170440/46162668-40e72b80-c289-11e8-8ba9-cf8ae6306f8b.png)

🚨 Too much emojis

underscope committed:
Updated login handling :wrench:

- Use learner app for login views
- Initialized learner app router & store

underscope committed:
:lipstick:

underscope committed:
Redirect admin upon login :wrench:

underscope committed:
Updated autocomplete icon :wrench:

underscope committed:
Cleanup

- Set key for Datatables
- Destructure scoped slots
- Handling error msg for enrollment modal
